### PR TITLE
Plot crud

### PR DESCRIPTION
**Pull request by ValzM**

### CODE REVIEW

File: app/persistence/plot_repository.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+def connect_db():
+    """
+    Connect to the database.
+
+    :return: Connection to the  database.
+    """
+    database_name = os.getenv("DATABASE_NAME")
+    database_host = os.getenv("DATABASE_HOST")
+    database_port = os.getenv("POSTGRES_PORT")
```
Comment (Eduardo-Cerqueira) [inline app/persistence/plot_repository.py]: DATABASE_PORT

File: app/persistence/plot_repository.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    :param (int) number: The plot number to delete.
+    """
+    conn = connect_db()
+    cur = conn.cursor()
+    cur.execute("DELETE FROM plot WHERE number = %s", (number,))
+    conn.commit()
+    conn.close()
+
+# Test CRUD
+if __name__ == "__main__":
```
Comment (Eduardo-Cerqueira) [inline app/persistence/plot_repository.py]: Remove this, this will be tested using the routes

**Review by Eduardo-Cerqueira — APPROVED** [review]

File: app/persistence/plot_repository.py (hunk 1 above)
Comment (ValzM) [reply app/persistence/plot_repository.py]: Done

File: app/persistence/plot_repository.py (hunk 2 above)
Comment (ValzM) [reply app/persistence/plot_repository.py]: Done

**Pull request merged**